File: LeoApp/app/src/main/java/de/slg/leoapp/utility/exception/NodeIndexOutOfBoundsException.java
package de.slg.leoapp.utility.exception;

public class NodeIndexOutOfBoundsException extends RuntimeException {
    public NodeIndexOutOfBoundsException() { super(); }
    public NodeIndexOutOfBoundsException(String message) { super(message); }
    public NodeIndexOutOfBoundsException(String message, Throwable cause) { super(message, cause); }
    public NodeIndexOutOfBoundsException(Throwable cause) { super(cause); }
}

File: core/src/main/java/de/leoapp_slg/core/datastructure/exception/NodeIndexOutOfBoundsException.kt
@file:Suppress("unused")

package de.leoapp_slg.core.datastructure.exception

class NodeIndexOutOfBoundsException : RuntimeException {
    constructor() : super()
    constructor(message: String) : super(message)
    constructor(message: String, cause: Throwable) : super(message, cause)
    constructor(cause: Throwable) : super(cause)
}
